packages/router/src/grpc/view-protocol-server/notes.test.ts

```diff
@@ -274,6 +270,7 @@ describe('Notes request handler', () => {
   });
 });
 
+// TODO: Regenerate new test data, these are not the right types
```

Comment:
Perhaps we should use `new SpendableNoteRecord()` instead of `.fromJson()` as it makes it easier to see which fields are invalid.

Reply:
i think using the constructor would be worse, unless we have a systematic way to create data. many of these fields are binary. i was able to just update the json